awx/ui/client/lib/theme/_variables.less

```diff
@@ -14,22 +14,24 @@
  * 1. Colors
  * 2. Typography
  * 3. Layout
+ * 4. Breakpoints
  *
  */
 
 // 1. Colors --------------------------------------------------------------------------------------
 
-@at-gray-light-3x: #fcfcfc;
-@at-gray-light-2-5x: #fafafa;
-@at-gray-light-2x: #f2f2f2;
-@at-gray-light: #ebebeb;
-@at-gray: #e1e1e1;
-@at-gray-dark: #d7d7d7;
-@at-gray-dark-2x: #b7b7b7;
-@at-gray-dark-3x: #A9A9A9;
-@at-gray-dark-4x: #848992;
-@at-gray-dark-5x: #707070;
-@at-gray-dark-6x: #161b1f;
+@at-gray-fc: #fcfcfc;
```

Comment:
Thanks for cleaning this up. I definitely prefer the more explicit names since there are so many shades of gray.